### PR TITLE
fix: modify the logic to validate the URL input

### PR DESCRIPTION
**Pull request by kimjinmyeong**
This PR resolves issue [#1388](https://github.com/lablup/backend.ai/issues/1388) in **backend.ai** repo.
Instead of the regular expression pattern, I added a function that uses the `URL` library to validate `url` input.

`_checkValidationMsgOnRegistryUrlInput` function is called when the **@click** or **@change** events are triggered. 
`setCustomValidity` method sets the `validationMessage` for the **mwc-textfield**. 
> **setCustomValidity** sets a custom validity message (also overwrites validationMessage). If this message is not the empty string, then the element is suffering from a custom validity error and does not validate.

See more details for mwc-textfield [here](https://npm.io/package/@material/mwc-textarea)

### CODE REVIEW

File: src/components/backend-ai-registry-list.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import '@material/mwc-icon-button';
 import '@material/mwc-textfield';
 import '@vaadin/grid/vaadin-grid';
 import {TextField} from '@material/mwc-textfield';
+import { TextFieldChangeEvent } from '@vaadin/text-field';
```
Comment (bezzang-dev) [inline src/components/backend-ai-registry-list.ts]: I removed this next commit

**Comment by yomybaby** [comment]
@kimjinmyeong  Please check the [comment](https://github.com/lablup/backend.ai-webui/pull/1806#issuecomment-1647346197) by cla-assistant.

**Comment by kimjinmyeong** [comment]
> @kimjinmyeong Please check the [comment](https://github.com/lablup/backend.ai-webui/pull/1806#issuecomment-1647346197) by cla-assistant.

@yomybaby 
Should the comment that says "CLA not signed yet" be changed to "CLA signed"? I changed the git config user.name and user.email to my lablup email and made a commit, but something seems to have gone wrong. Even after signing and rechecking, it's still not working...

**Comment by yomybaby** [comment]
@kimjinmyeong Please check [this](https://docs.github.com/ko/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account). Please check the your @lablup.com email on the Github account email list.

**Comment by kimjinmyeong** [comment]
@yomybaby 
Even after adding @lablup.com email to the email list and signing again, it's not getting updated..
Should I just close this PR and create a new one?

**Review by yomybaby — APPROVED** [review]
LGTM! Thank you for your contribution.

**Pull request merged**